amlip_docs/rst/demo/wan.rst

```diff
@@ -0,0 +1,189 @@
+.. include:: ../exports/alias.include
+
+.. _demo_inference:
+
+########
+WAN Demo
+########
+
+This demonstrator shows how to implement 2 kind of nodes:
+:ref:`user_manual_nodes_inference` and :ref:`user_manual_nodes_edge`.
+With these 2 nodes implemented, the user can deploy as many nodes of each kind as desired and check the
+behavior of a simulated |amlip| network running.
+They are implemented one in Python and one in C++ to demonstrate as well how to instantiate each kind of node
+with different :term:`APIs <API>`, and to prove the communication between the 2 implementations.
+
+Simulation
+==========
+
+TensorFlow
+----------
+
+TensorFLow is an end-to-end machine learning platform with pre-trained models.
+
+Darknet
+-------
+
+Darknet is an open source neural network framework written in C and CUDA with pre-trained models.
+
+Edge Node
+---------
+
+This node simulates a :ref:`user_manual_nodes_edge`.
+It is implemented in |python| using :code:`amlip_py` API.
+There are 2 different ways to run it, an automatic one and a manual one:
+
+Inference Node
+--------------
+
+This node simulates a :ref:`user_manual_nodes_inference`.
+It is implemented in |cpp| using :code:`amlip_cpp` API.
+
+To run it, one string argument is required. This will be the model this node will use to make the inference.
+
+Installation
+============
+
+First of all, check that :code:`amlip_demo_nodes` sub-package is correctly installed.
+If it is not, please refer to :ref:`developer_manual_installation_sources_linux_colcon_demos`.
+
+Requirements
+============
+
+The demo requires the following tools to be installed in the system:
+
+.. code-block:: bash
+
+    sudo apt install -y  swig espeak alsa-utils libopencv-dev ffmpeg
+    pip3 install -U pyttsx3 opencv-python tensorflow tensorflow-hub tensorflow-object-detection-api pygame
+
+Get the models
+==============
+
+Using TensorFlow
+----------------
+
+.. code-block:: bash
+    cd ~/AML-IP-demos-ws/src/AML-IP-demos/amlip_demos/resource/tensorflow/models/
```

Comment:
This is not shown, and produces an error. Left a blank line between sphinx tab and its body
```suggestion
.. code-block:: bash

    cd ~/AML-IP-demos-ws/src/AML-IP-demos/amlip_demos/resource/tensorflow/models/
```